lib/studies/reply/adaptive_nav.dart

```diff
@@ -260,7 +271,7 @@ class _NavigationRailHeader extends StatelessWidget {
                         ).animate(animation),
                         child: const Icon(
                           Icons.arrow_left,
-                          color: ReplyColors.blue50,
+                          color: ReplyColors.white50,
```

Comment:
Should these be conditional on isDark? or is white preferred in both light and dark?

Reply:
White is preferred for both.